migrations/20241030234724_first_migration.sql

```diff
@@ -0,0 +1,21 @@
+-- +goose Up
+-- create "metrics" table
+CREATE TABLE "metrics" (
+  "id" character varying(255) NOT NULL,
+  "created_at" timestamptz NULL,
+  "updated_at" timestamptz NULL,
+  "deleted_at" timestamptz NULL,
+  "type" text NOT NULL,
+  "m_source" text NULL,
+  "delta" bigint NULL,
```

Comment:
[CLARIFICATION] Why don't you want to split data?

Reply:
What do you mean split the data? store data and metrics in different tables?

Reply:
We have two types of data. So, it seems you need to normalize the database schema.
https://www.freecodecamp.org/news/database-normalization-1nf-2nf-3nf-table-examples/

Reply:
transformed into 3rd normal form